src/utils/notify_props.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {Preferences} from 'constants';
+
+export function getEmailInterval(enableEmailNotification, enableEmailBatching, emailIntervalPreference) {
```

Comment:
This is added to be used for mobile RN and webapp.  Currently on webapp, we have this https://github.com/mattermost/mattermost-webapp/blob/master/utils/utils.jsx#L1554.